FWCore/Concurrency/interface/include_first_syncWait.h

```diff
@@ -0,0 +1,40 @@
+#ifndef FWCore_Concurrency_syncWait_h
+#define FWCore_Concurrency_syncWait_h
+//
+//  syncWait.h
+//
+//  This file must be included before any other file that include tbb headers
+//
+//  Created by Chris Jones on 2/24/21.
+//
+#define TBB_PREVIEW_RESUMABLE_TASKS 1
```

Comment:
Hi @Dr15Jones, we suspect this define is causing a recent CXXMODULES IB regression. Would it be possible to move this define (I saw another one in the cc file) to the build system and possibly add it to the rootcling invocation (if scram does not do that automatically).


cc: @davidlange6

Reply:
> in one of the headers including tbb/task_group.h but not consistently.

That was intentional. It is only turned on where the feature is needed. Again, that was the recommendation of the tbb developers.

Reply:
@Dr15Jones , yes we can set it at build time. @davidlange6 , are you testing it by adding it in the tbb.xml toolfile? You can add it in the toolfile and then scram should set itfo rboth normal build and rootcling

Reply:
this macro is now defined via tbb toolfile ( see https://github.com/cms-sw/cmssw/pull/33140 and https://github.com/cms-sw/cmsdist/pull/6713 )

Reply:
> > in one of the headers including tbb/task_group.h but not consistently.
> 
> That was intentional. It is only turned on where the feature is needed. Again, that was the recommendation of the tbb developers.

I'd assume they did not know about the fact we use C++ modules in the dictionaries. In theory, we can still use the macro the way we do, and only pass the relevant `-D` to rootcling. Not sure if that's worth it.